CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.6.2)
```

Comment:
What do you think is the right trade off:

1. Keep min version and CI version the same, and expect users to have this same version installed => which ensures no version related bugs
2. Set min version to 3.6.0 and ignore the patch number so that users can install any 3.6.x version for convenience => which risks hitting version related bugs